package.json

```diff
@@ -11,6 +11,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.1",
+    "react-router-dom": "^6.5.0",
```

Comment:
Great to make the installation of the react router "with the latest version" which is necessary for routing inside our website!